source/conf.py

```diff
@@ -61,6 +61,7 @@
 mongodb42recommended = '4.2.17-17'
 pmm2recommended = '2.24.0'
 gkerecommended = '1.22'
+certmanagerrecommended = '1.6.1'
 
```

Comment:
@egegunes why not v1.7.1?

Reply:
we will put here most recent versions in a separate PR (the one where we update supported platforms)